### ISSUE TITLE
Radio button accessibility

### ISSUE THREAD

**Issue by hartmans** [PATCH-UPGRADE/whs#120]
The radio buttons on the device creation page are missing aria labels/binding to the labels they have, so the screen reader does not announce which option is currently selected.
